python/veles/schema/fields.py

```diff
@@ -283,3 +283,24 @@ def _load(self, value):
 
     def _dump(self, value):
         return six.text_type(value.name)
+
+
+class Exception(Field):
```

Comment:
Why did you do it like this instead of `Object` with `value_type` set to `VelesException` and make `VelesException` be a model (albeit a very custom one)? Then you wouldn't need to do all this additional validation here.